neuro_san/test/assessor/assessor.py

```diff
@@ -28,8 +28,6 @@ class Assessor:
     to an input test case.
     """
 
-    SHIPPED_HOCONS = FileOfClass(__file__, path_to_basis="../../registries")
```

Comment:
No need for ../.. reach-around any more, which is problematic when source like this moves around.